app/main.py

```diff
@@ -1 +1,38 @@
-#  write your code here
+class Animal:
+    def __init__(self, name: str, appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.appetite == 0 or not self.is_hungry:
```

Comment:
`if not self.is_hungry` could probably be enough here. Consider to remove a first part of the condition